fs/xfstests.py

```diff
@@ -453,6 +454,8 @@ def test(self):
                 failures = True
 
         else:
+            if self.group:
+                self.error("change group param to null in yaml to run test range")
```

Comment:
this condition can be moved immediatly after
self.group = self.params.get('group', default=None)
self.test_range = self.params.get('test_range', default=None)

So it can cancel test early 

also can you change yaml test_range: "null"    to just test_range: 

as null inside double quote becomes a string, also the default value in setup is given None

Reply:
I tried with the suggested changes, test fails with below error because of using unconditional userdel in teardown.

`224         self.group = self.params.get('group', default=None)
225         self.test_range = self.params.get('test_range', default=None)
226         self.scratch_mnt = self.params.get(
227             'scratch_mnt', default='/mnt/scratch')
228         self.test_mnt = self.params.get('test_mnt', default='/mnt/test')
229         self.disk_mnt = self.params.get('disk_mnt', default='/mnt/loop_device')
230         self.fs_to_test = self.params.get('fs', default='ext4')
231         self.run_type = self.params.get('run_type', default='distro')
232 
233         if self.group and self.test_range:
234                 self.error("change group param to null in yaml")
`

ERROR: Command 'userdel -f 123456-fsgqa' failed.\nstdout: b''\nstderr: b"userdel: user '123456-fsgqa' does not exist

Reply:
will make changes in yaml for null